feature-ledger-impl/src/main/java/io/novafoundation/nova/feature_ledger_impl/presentation/account/connect/generic/selectLedger/SelectLedgerGenericImportViewModel.kt

```diff
@@ -37,12 +35,13 @@ class SelectLedgerGenericImportViewModel(
     override suspend fun verifyConnection(device: LedgerDevice) {
         ledgerMessageCommands.value = LedgerMessageCommand.Hide.event()
 
-        val universalAccount = substrateApplication.getUniversalAccount(device, confirmAddress = false)
-        val payload = PreviewImportGenericLedgerPayload(
-            account = universalAccount.toGenericParcel(),
-            deviceId = device.id
+        val payload = SelectLedgerAddressPayload(
+            deviceId = device.id,
+            chainId = getPreviewBalanceChainId()
         )
```

Comment:
I think we can keep payload the same but pass chainId of Polkadot to show amount in DOT